src/main/java/com/notably/commons/path/AbsolutePath.java

```diff
@@ -113,6 +123,7 @@ public int compareTo(AbsolutePath path) {
 
     @Override
     public boolean equals(Object object) {
+        requireNonNull(object);
```

Comment:
This should be removed! Usually, `equals` still accepts `null` inputs. It'll just return false if that's the case!